js/deribit.js

```diff
@@ -388,26 +388,28 @@ module.exports = class deribit extends Exchange {
     }
 
     async fetchStatus (params = {}) {
-        const request = {
-            // 'expected_result': false, // true will trigger an error for testing purposes
-        };
-        const response = await this.publicGetTest (this.extend (request, params));
+        const response = await this.publicGetStatus (params);
         //
         //     {
-        //         jsonrpc: '2.0',
-        //         result: { version: '1.2.26' },
-        //         usIn: 1583922623964485,
-        //         usOut: 1583922623964487,
-        //         usDiff: 2,
-        //         testnet: false
+        //         "jsonrpc": "2.0",
+        //         "result": {
+        //             "locked": "false" // true, partial, false
+        //         },
+        //         "usIn": 1650641690226788,
+        //         "usOut": 1650641690226836,
+        //         "usDiff": 48,
+        //         "testnet": false
         //     }
         //
-        this.status = this.extend (this.status, {
-            'status': 'ok',
-            'updated': this.milliseconds (),
+        const result = this.safeString (response, 'result');
+        const locked = this.safeString (result, 'result') !== 'false';
```

Comment:
This line is not transpileable – break it down into multiple lines.

Reply:
i've changed to `===` instead of `!==`, it should transpile now.